packages/cli/src/ui/commands/mcpCommand.ts

```diff
@@ -248,10 +249,16 @@ const getMcpStatus = async (
       if (serverTools.length > 0) {
         message += '\n';
       }
-      message += `  ${COLOR_CYAN}Prompts:${RESET_COLOR}\n`;
-      serverPrompts.forEach((prompt: DiscoveredMCPPrompt) => {
+      message += `  ${RESET_COLOR}Prompts:${RESET_COLOR}\n`;
+      serverPrompts.forEach((prompt) => {
+        const originalName = prompt.originalName || prompt.name;
```

Comment:
I think this would make more sense as `prompt.name` the name the prompt has in the MCP server and `prompt.displayName` as the potentially conflict-aware display name